lnclient/lnd/lnd.go

```diff
@@ -359,7 +359,11 @@ func (svc *LNDService) SendKeysend(ctx context.Context, amount uint64, destinati
 
 	destCustomRecords := map[uint64][]byte{}
 	for _, record := range custom_records {
-		destCustomRecords[record.Type] = []byte(record.Value)
+		decodedValue, err := hex.DecodeString(record.Value)
+		if err != nil {
+			return "", err
+		}
+		destCustomRecords[record.Type] = decodedValue
```

Comment:
LND actually accepts keysend record as a hex encoded value (see [this](https://docs.lightning.engineering/lightning-network-tools/lnd/send-messages-with-keysend#send-a-message-to-other-nodes))

Reply:
@im-adithya you linked to the cli docs. The LND DestCustomRecords is a `map[uint64][]byte`

Reply:
Yup, I tried it from the CLI in polar, and what we have right now does it fine. Because after passing the hex encoded in the cli, it still returned the same thing while listing transactions (the hex encoded value `48617070792047656E6573697320426C6F636B2044617921 ` but not the actual string `Happy Genesis Block Day!`)

Reply:
@im-adithya I don't understand, why are you testing the cli?

Reply:
Discard above, it accepts a hex encoded value in bytes and the above line does exactly that. tACK ✅

Reply:
Just to confirm - it accepts the value as *bytes* not as hex-encoded bytes - that's why the line above decodes the hex first

Reply:
Yup got that, thanks!